client/src/Components/Common/homePage/style.js

```diff
@@ -0,0 +1,35 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        maxWidth: 400,
+        flexGrow: 1,
+      },
+    category:{
+        marginBottom:30,
+        marginTop:20,
+    },
+    text:{
+        marginRight:20,
+    },
+    textDiv:{
+        display: "flex",
+        justifyContent: "space-between",
+        paddingLeft: 20,
+    },
+    galleryDiv:{
+        display: "flex",
+        justifyContent: "flex-end",
+        flexDirection: "column",
+        marginBottom: 30,
+    },
+    customizeDiv:{
+        display: "flex",
+        justifyContent: "center",
+        flexDirection: "column",
+        marginBottom:30,
+    },
+   
+}))
```

Comment:
Really like the roundness of the numbers here. Whenever I see values like `237px` I wonder "where did that value come from?" but this is awesome 👍